sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventDataBatchBase.java

```diff
@@ -106,7 +106,7 @@ public int getSizeInBytes() {
      * @throws IllegalArgumentException if {@code eventData} is {@code null}.
      * @throws AmqpException if {@code eventData} is larger than the maximum size of the {@link EventDataBatch}.
      */
-    public boolean tryAdd(final EventData eventData) {
+    boolean tryAdd(final EventData eventData) {
         if (eventData == null) {
             throw logger.logExceptionAsWarning(new IllegalArgumentException("eventData cannot be null"));
```

Comment:
This should be an NPE rather than IllegalArgument. https://azure.github.io/azure-sdk/java_implementation.html#java-errors-system-errors

Reply:
fixed, thanks Connie